README.md

```diff
@@ -1,13 +1,16 @@
 # Project Auth API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The aim of this project was to build an API with authentication to implement a registration flow, and a frontend with forms to register, sign in, and view some content once you're logged in.
 
 ## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We followed the code-along/live sessions quite to closely, and managed to build project from that. We worked in branches throughout the whole projects. The biggest problem was getting the connection to the database in Atlas to work, and we found that one connection worked while the other didn't. 
+If we had more time, we would work on styling the frontend and making it more user friendly. We could refactor the code alot, and make it shorter. There are some small bugs that we would fix as well.
+Tools used for this project: React, JavaScript, Redux, HTML5, CSS, Styled Components, Express API, MongoDB, Mongoose.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Back-end:
+https://project-auth-d77e5zoeyq-lz.a.run.app/
```

Comment:
Nice job with the project! Seems to match the requirements for this week. The frontend looks nice as well :) 

The single bug that we can find in your frontend is that after registration you route to "page not found".

/Michelle & Emilia